### PR TITLE
Fix double modications

### PR DESCRIPTION
**Pull request by mschwoer**
Fix the issues discovered here https://github.com/MannLabs/alphabase/pull/186
by removing the double-bookeeping in the modifications.

@jalew188 and me decided to break the public API (the methods in `modifications.py`) but not the reading of speclib files
This is how is looks like in alphaDIA:
image

Please check very carefully this breaks something you know about

### CODE REVIEW

**Review by jalew188 — APPROVED** [review]
LGTM.

I saw unimod_to_tsv.ipynb was excuted, did you overwrite modification.tsv?

File: alphabase/constants/const_files/__used_mod.yaml [hunk 1]
```diff
@@ -10,23 +10,23 @@ GlyGly@K:
   unimod_id: 121
 
 #unimod modifications
```
[the remark this answers is not in the document]
Comment (mschwoer) [reply alphabase/constants/const_files/__used_mod.yaml]: thanks, deleted it.. what about `__emass_element.yaml`?

**Comment by mschwoer** [comment]
> LGTM.
> 
> I saw unimod_to_tsv.ipynb was excuted, did you overwrite modification.tsv?

yes I did

File: alphabase/constants/const_files/__used_mod.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jalew188) [reply alphabase/constants/const_files/__used_mod.yaml]: Just keep it. Different regions on the earth may have different isotope distributions for a same atom. Currently we used NIST settings.

**Review by GeorgWa — APPROVED** [review]
LGTM, this was long necessary. 
Will be breaking with the default yaml.

**Comment by jalew188** [comment]
@mschwoer @GeorgWa ready to merge!!

File: alphabase/constants/const_files/psm_reader.yaml [hunk 2]
```diff
@@ -52,10 +52,10 @@ maxquant:
       - 'K(Dimethyl)'
     'Dimethyl@R':
       - 'R(Dimethyl)'
-    'Dimethyl@Any N-term':
+    'Dimethyl@Any_N-term':
       - '(Dimethyl)'
-    'Acetyl@Protein N-term':
-      - '_(Acetyl (Protein N-term))'
+    'Acetyl@Protein_N-term':
+      - '_(Acetyl (Protein_N-term))'
```
Comment (jalew188) [inline alphabase/constants/const_files/psm_reader.yaml:58]: The actual unimod names contain a space, so we cannot modify the MQ source PTM names

Comment (mschwoer) [reply alphabase/constants/const_files/psm_reader.yaml:58]: release in 1.3.0, fixed here for 1.4.2: https://github.com/MannLabs/alphabase/pull/241

**Pull request merged**